src/__tests__/book-screen.js

```diff
@@ -22,9 +22,9 @@ async function renderBookScreen({user, book, listItem, route} = {}) {
   if (book === undefined) {
     book = await booksDB.create(buildBook())
   }
-  if (listItem === undefined) {
+  if (listItem == null) {
```

Comment:
Since `req.query` (`Record<string, string>`) has been changed to `req.url.searchParams` (`URLSearchParams`), its behavior also changed. When requesting a non-existing search parameter, `URLSearchParams` returns an explicit `null`.

Leaving it with a weak comparison (`==`) so that the logic that previously relied on `undefined` keeps working.